src/vs/monaco.d.ts

```diff
@@ -7455,6 +7455,8 @@ declare namespace monaco.languages {
 		TypeParameter = 25
 	}
 
+	export function getAriaLabelForSymbol(symbolName: string, kind: SymbolKind): string;
```

Comment:
not sure why this is needed

Reply:
I think the 

```
/**
 * @internal
 */
```

comment will prevent this. But you shouldn't add to this file unless you are trying to ship this function with monaco